Dockerfile

```diff
@@ -40,6 +39,28 @@ RUN cd nginx && \
 # Production container starts here
 FROM zappi/nginx:1.25.3
 
+USER root
+
+# Set up official Nginx repository
+RUN apt-get update -y && \
+    apt-get install --no-install-recommends -y \
+        ca-certificates \
+        gnupg2 \
+        wget && \
+    rm -rf /var/lib/apt/lists/* /tmp/* && \
+    wget -q -O - https://nginx.org/keys/nginx_signing.key | gpg --dearmor > /etc/apt/keyrings/nginx.gpg && \
+    echo "deb [signed-by=/etc/apt/keyrings/nginx.gpg] http://nginx.org/packages/mainline/debian bookworm nginx" | tee /etc/apt/sources.list.d/nginx.list && \
+    apt-get purge --auto-remove -y \
+        ca-certificates \
+        gnupg2 \
+        wget
+
+# Module: OpenTelemetry (OTel)
+RUN apt-get update -y && \
+    apt-get install --no-install-recommends -y \
+      nginx-module-otel && \
```

Comment:
I see the [installations docs](https://docs.nginx.com/nginx/admin-guide/dynamic-modules/opentelemetry/) linked has `apt-get install nginx-plus-module-otel` for Ubuntu. Is this the same thing under a different package name?

Reply:
THat's the package for Nginx Plus, we're using the OSS version. OSS don't have great docs so you'll notice a lot of referencing to the Plus docs where it make sense.

Reply:
Ah, I see.